src/Components/Main/Banners.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import "./../../styles/slider-styles.css";
+import BtnSlider from "./BtnSlider.jsx";
+
+const Banners = (props) => {
+  const [slideIndex, setSlideIndex] = useState(1);
+
+  const prevSlide = () => {
+    if (slideIndex > 0) {
+      setSlideIndex(slideIndex);
+    } else if (slideIndex === 0) {
+      setSlideIndex(props.size);
+    }
+  };
```

Comment:
the `if/else` could be refactored, if the `slideIndex` is greater than zero you update the `slideIndex` if it is not greater than zero you also update the `slideIndex`, what is changing here is not the action that is being executed is the value that is being passed to the function

```suggestion
  const prevSlide = () => {
    setSlideIndex(prevIndex => prevIndex > 0 ? prevIndex : props.size);
  };
```